src/hooks/usePopoverRef/useHoverPopoverRef.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as styles from './styles.less';
+import PopoverContainer, { PopoverContainerProps } from './components/Container';
+import {
+    useHoverToggleController,
+    useRelativePositioning,
+    useOverlayPortal,
+} from '@equinor/fusion-components';
+
+export default <T extends HTMLElement>(
```

Comment:
The only difference between usePopoverRef and useHoverPopoverRef is the usage of useHoverToggleController vs. useClickToggleController. Perhaps this could be merged into the usePopoverRef hook with an option for hover/click?